docs/source/data_recipes/ERA5_preprocessing_example.md

```diff
@@ -100,56 +97,78 @@ variables according to the Virtual Rainforest naming convention (see
-temperatures; in the future, this should be done for each year.
+In addition to the variables from the ERA5-Land datasset, a time series of atmospheric
+$\ce{CO_{2}}$ is needed. We add this here as a constant field. Mean annual temperature
+is calculated from the full time series of air temperatures; in the future, this should
+be done for each year.
 
 ```{code-cell} ipython3
-dataset_simple = dataset_renamed.drop_vars(
-  [
-    'wind_speed_ref',
-    'soil_moisture_ref',
-    'top_soil_temperature_ref',
-    'TOA_incoming_solar_radiation',
-    'total_could_cover',
-  ]
-)
-dataset_simple['atmospheric_co2_ref'] = DataArray(
-  np.full_like(dataset_simple['air_temperature_ref'],400),
+dataset_renamed['atmospheric_co2_ref'] = DataArray(
+  np.full_like(dataset_renamed['air_temperature_ref'],400),
   dims=['time', 'latitude', 'longitude'],
 )
-dataset_simple['mean_annual_temperature'] = (
-  dataset_simple["air_temperature_ref"].mean(dim = 'time')
+dataset_renamed['mean_annual_temperature'] = (
+  dataset_renamed["air_temperature_ref"].mean(dim = 'time')
   )
-dataset_simple
+dataset_renamed.data_vars
+```
+
+### 7. Change coordinates to x-y in meters
+
+The following code segment changes the coordinate names from `longitude/latitude` to
+`x/y` and the units from `minutes` to `meters`. The ERA5-Land coordinates are treated as
+the centre points of the grid cells which means that when setting up the grid, an offset
+of 4.5 km has to be added.
+
+```{code-cell} ipython3
+dataset_xy = dataset_renamed.rename_dims({'longitude':'x','latitude':'y'}).assign_coords({'x':np.arange(0,180000,9000),'y':np.arange(0,180000,9000)}).drop({'longitude','latitude'})
+dataset_xy.coords
+```
+
+### 8. Scale to 90 m resolution
+
+The Virtual Rainforest is run on a 90 x 90 m grid. This means that some form of spatial
+scaling has to be applied to the dataset, **which is not yet implemented!**
+
+For the purpose of a dummy simulation in the development stage, the coordinates can be
+overwritten to match the Virtual Rainforest grid and we can select a smaller area.
+When setting up the grid, an offset of 45 m has to be added. Note that the resulting
+dataset does no longer match a digital elevation model for the area!
+
+```{code-cell} ipython3
+dataset_xy_100 = dataset_renamed.rename_dims({'longitude':'x','latitude':'y'}).assign_coords({'x':np.arange(0,1800,90),'y':np.arange(0,1800,90)}).drop({'longitude','latitude'})
+dataset_xy_dummy = dataset_xy_100.isel(x=np.arange(9),y=np.arange(9))
+dataset_xy_dummy.coords
+```
+
+### 9. Add time_index
+
+The dummy model iterates over time indices rather than real datetime. Therefore, we add
+a `time_index` coordinate to the dataset:
+
+```{code-cell} ipython3
+dataset_xy_timeindex = dataset_xy_dummy.rename_dims({'time':'time_index'}).assign_coords({'time_index':np.arange(0,24,1)})
+dataset_xy_timeindex.coords
 ```
 
-### 6. Save netcdf
+### 10. Save netcdf
 
```

Comment:
I'd add a further section giving the grid config `TOML` to load this data, which can link back to your comment about the offset.

```toml
[core.grid]
cell_nx = 9
cell_ny = 9
cell_area = 8100
xoff = -45.0
yoff = -45.0
```

Reply:
I added a section at the end and referred to it where I mention the offset. I also added a link to the config section in the VR setup.